frame/support/procedural/src/construct_runtime/expand/executive.rs

```diff
@@ -0,0 +1,107 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+use crate::construct_runtime::parse::ExecutiveSection;
+use frame_support_procedural_tools::generate_crate_access_2018;
+use proc_macro2::{Ident, TokenStream};
+use quote::quote;
+use syn::Result;
+
+pub fn expand_executive(
+	runtime: &Ident,
+	system: &Ident,
+	scrate: &TokenStream,
+	block: &TokenStream,
+	executive_section: ExecutiveSection,
+) -> Result<TokenStream> {
+	let executive = generate_crate_access_2018("frame-executive")?;
```

Comment:
This will play nicely with the new `frame` crate, it should re-export the `frame-executive`.